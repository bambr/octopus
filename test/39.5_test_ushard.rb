#!/usr/bin/ruby
# coding: utf-8

$: << File.dirname($0)
require '39_test_ushard'


$one.create_shard 1, :POR, "one"

# change shard type
log_try { $one.create_shard 1, :PAXOS, "one" }

