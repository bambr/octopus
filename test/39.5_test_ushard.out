# one.ping()
:pong

# two.ping()
:pong

# one.create_shard(1, :POR, "one")
:success

# one.create_shard(1, :PAXOS, "one")
Failed with: {code: 0x202, message: 'can't change shard type'}
